src/Swift.Bindings/src/Marshaler/NameProvider.cs

```diff
@@ -7,10 +7,10 @@ namespace BindingsGeneration;
 /// <summary>
 /// Represents a generic parameter name mapping.
 /// </summary>
-/// <param name="PlaceholderName">The name of the generic type parameter e.g. T0.</param>
+/// <param name="TypeParameter">The name of the generic type parameter e.g. T0.</param>
 /// <param name="MetadataName">The name of the metadata type parameter.</param>
 /// <param name="PayloadName">The name of the payload type parameter. </param>
-public record struct GenericParameterCSName(string PlaceholderName, string MetadataName, string PayloadName);
+public record struct GenericParameterCSName(string TypeParameter);
```

Comment:
Should these two get removed since the signature only accepts `string TypeParameter` now?